backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -950,8 +955,13 @@ public Observation createAOEObservation(String uniqueName, CodeableConcept code,
 
   private CodeableConcept createLoincConcept(String codingCode, String codingDisplay, String text) {
     var concept = new CodeableConcept().setText(text);
+    var coding = concept.addCoding();
 
-    concept.addCoding().setSystem(LOINC_CODE_SYSTEM).setCode(codingCode).setDisplay(codingDisplay);
+    coding.setSystem(LOINC_CODE_SYSTEM).setCode(codingCode);
+
+    if (StringUtils.isNotBlank(codingDisplay)) {
+      coding.setDisplay(codingDisplay);
```

Comment:
is this just an extra null check or does this relate somehow to the long name addition? want to make sure I'm not missing something conceptually if the later

Reply:
I don't think we have any other null checks for this value (we just check the object it comes from), so not necessarily extra. 

But also per the issue we don't want to add the value to the fhir bundle if it is null / empty.